
export function handleError(error: unknown) {
    console.log(error);
}
